submissions/akinbobolaFaith/dom/signUp.js

```diff
@@ -0,0 +1,122 @@
+let signupForm = document.querySelector('.information'),
+    firstName = document.getElementById('fname'),
+    lastName = document.getElementById('lname'),
+    phoneNumber = document.getElementById('pnumber'),
+    email = document.getElementById('email'),
+    password = document.getElementById('password'),
+    confirmPassword = document.getElementById('cpassword')
+;
+
+function important (field, event) {
+    if(field.value == '') {
+        field.nextElementSibling.innerHTML = "This field is required";
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function firstNameValidation(event) {
+    let firstNameRegex = /^[a-z]+$/i;
+    if(!firstName.value.match(firstNameRegex)) {
+        firstName.nextElementSibling.innerHTML = "Please enter your First Name";
+        event.preventDefault();
+        return false;
+    }  else {
+        firstName.nextElementSibling.innerHTML = "";
+        return true;
+    }
+} 
+
+function lastNameValidation(event) {
+    let lastNameRegex = /^[a-z]+$/i;
+    if(!lastName.value.match(lastNameRegex)) {
+        lastName.nextElementSibling.innerHTML = "Please enter your Last Name";
+        event.preventDefault();
+        return false;
+    } else {
+        lastName.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function phoneNumberValidation(event) {
```

Comment:
Also, phonevalidation, email and password are practically the same thing as well. Pass the element name, regex and error messages as variables to one function @faithakinbobola